pkg/console/config/config.go

```diff
@@ -1,169 +1,136 @@
-package tlsconfig
+package config
 
 import (
 	"crypto/tls"
 	"errors"
 	"fmt"
 	"os"
+	"path/filepath"
+	"strconv"
 	"strings"
 	"sync"
-	"sync/atomic"
 
-	"github.com/fsnotify/fsnotify"
 	ocpconfigv1 "github.com/openshift/api/config/v1"
 	"github.com/openshift/library-go/pkg/crypto"
 	"k8s.io/apimachinery/pkg/util/yaml"
 	"k8s.io/client-go/util/cert"
 	"kubevirt.io/client-go/log"
 
 	"github.com/kubevirt/vm-console-proxy/api/v1alpha1"
+	"github.com/kubevirt/vm-console-proxy/pkg/filewatch"
```

Comment:
Did you follow any specific rule to separate the imports? E.g. First block for native go libs, k8s imports, third OpenShift, etc.? Same question for the rest of the files.

Reply:
Yes:
- Standard library
- Imports from modules in `go.mod`
- Local imports from this project

But my IDE does not enforce it, so sometimes imports are not separated to these blocks.